components/config-mgmt-service/integration_test/event_counts_test.go

```diff
@@ -659,17 +659,6 @@ func TestEventTypeCountsProjectFilter(t *testing.T) {
 	}
 }
 
-func findEventCountByName(eventName string,
```

Comment:
This function is here also https://github.com/chef/automate/blob/3ebc63c6506f4d570626a057594ef1a87e53d847/components/config-mgmt-service/integration_test/event_task_counts_test.go#L707